java/README.md

```diff
@@ -10,34 +10,41 @@ The Semantic Kernel for Java is an SDK that implements the key concepts of the S
 to be used in Java applications and services in an idiomatic way, and to be easily integrated with other Java libraries
 and frameworks.
 
-## Get Started
+## Get started
 
 To run the LLM prompts and semantic functions in this kernel, make sure you have
 an [Open AI API Key](https://openai.com/api/)
 or [Azure Open AI service key](https://learn.microsoft.com/azure/cognitive-services/openai/quickstart?pivots=rest-api).
 
 ### Requirements
 
-To build the semantic kernel, you will need to have:
+To build the project, you will need to have:
 
-- **Required**:
-    - [JDK 8](https://microsoft.com/openjdk/) or newer to build the Semantic Kernel library
-    - [JDK 17](https://microsoft.com/openjdk/) or newer to run samples
+- **Required**: 
+  - [JDK 8](https://microsoft.com/openjdk/) or newer to build the Semantic Kernel library
+  - [JDK 17](https://microsoft.com/openjdk/) or newer to run samples
+  - Azure OpenAI SDK for Java _(see below for instructions)_
```

Comment:
this is installed by default, not required

Reply:
Missed that line.

Reply:
Fixed in 37c9e30.